[TW-343 (#4018)
postmanlabs/postman-docs: src/pages/docs/administration/scim-provisioning/scim-provisioning-overview.md

    > The newly added user will have the developer role in Postman by default. You can later [update account roles in Postman](/docs/administration/managing-your-team/managing-your-team/#managing-roles).

* **Update user information**:

    * **Update user attributes**: Updates a user’s first and last name in Postman.

    * **Activate user**: Creates a new user on your Postman team, if one doesn't already exist, and activates the user to authenticate into your Postman team.

    * **Deactivate user**: Removes a user from your Postman team and deactivates their account, blocking the account from authenticating into Postman.

        > User accounts and the data corresponding to them won't be deleted. To permanently delete a user account and its data, [contact Postman support](https://www.postman.com/support/).

    * **Reactivate user**: Reactivates an existing deactivated user by unblocking the account's authentication into Postman and adds the account back to your Postman team.

> * **Create group**: Creates a new [user group](/docs/administration/managing-your-team/user-groups/) in Postman. When you assign the Postman app to a group, Postman creates a new account for each group member, adds each account to your organization's Postman team, and activates authentication for each user. If an account with a matching email ID to a group member exists, an [email invite](/docs/administration/managing-your-team/managing-your-team/#invites) to join your Postman team is sent to that user. Once the user accepts the invite, they'll be added to your team.

Reviewer: ```suggestion
* **Create group**: Creates a new [user group](/docs/administration/managing-your-team/user-groups/) in Postman. When you assign the Postman app to a group, Postman creates a new account for each group member, adds each account to your organization's Postman team, and activates authentication for each user. If an existing Postman account uses an email that matches a group member's email ID, an [email invite](/docs/administration/managing-your-team/managing-your-team/#invites) to join your Postman team is sent to that user. Once the user accepts the invite, they'll be added to your team.
```

    > Newly created groups will have the developer role in Postman by default. You can later [update group roles in Postman](/docs/administration/managing-your-team/managing-your-team/#managing-roles).

* **Delete group**: Deletes a user group in Postman. User accounts that were part of the deleted group are deactivated in Postman.

* **Update group information**:

    * **Update group attributes**: Updates a group's name in Postman.

    * **Update group members**: Adds or removes members from a group in Postman.

> Currently, Postman doesn't support the following provisioning features:

Reviewer 2: Don't say "Currently" here. Just say "Postman doesn't support..."

* Push groups from Postman to your Identity Provider (IdP)

> * Push/sync password updates

Reviewer 3: ```suggestion
* Push or sync password updates
```

> * Push/sync password updates

Reviewer: ```suggestion
* Push and sync password updates
```

1. Name your key and click **Generate**.
1. Copy your new API key for later use and click **Done**.

> You can revisit this page to manage your SCIM API keys. If you regenerate an existing API key, you will have the option to keep the first key active while you switch over.

> To continue enabling SCIM provisioning, see [Configuring SCIM with the SCIM API](#configuring-scim-with-the-scim-api), [Configuring SCIM with Okta](/docs/administration/scim-provisioning/configuring-scim-with-okta/), or [Configuring SCIM with Azure AD](/docs/administration/scim-provisioning/configuring-scim-with-azure-ad/). For further information or help configuring SCIM, [contact Postman support](https://www.postman.com/support/).

Reviewer 4: Do we have a document for OneLogin as well? If yes, let us refer to it here as well.
